tests/framework/utils_cpuid.py

```diff
@@ -54,7 +54,8 @@ def get_cpu_model_name():
     raw_cpu_model = info[1].strip()
     if platform.machine() == "x86_64":
         return raw_cpu_model
-    return CPU_DICT[CpuVendor.ARM][raw_cpu_model]
+    name = CPU_DICT[CpuVendor.ARM].get(raw_cpu_model)
+    return name if name else "Unknown"
```

Comment:
I think we can use something like below :
`name = CPU_DICT[CpuVendor.ARM].get(raw_cpu_model, "Unknown")`

This will return "Unknown" as default value if raw_cpu_model key is not found.

Reply:
Nice, updated.